src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -232,3 +232,22 @@
     </div>
   </div>
 {% endmacro %}
+
+{% macro multistore_switch(form, extraVars) %}
+  {% import '@PrestaShop/Admin/macros.html.twig' as self %}
+  {% if form.shop_restriction_switch is defined %}
+    {% set defaultLabel %}
+      <strong>{{ 'Check / Uncheck all'|trans({}, 'Admin.Actions') }}</strong> <br>
+      {{ 'You are editing this page for a specific shop or group. Click "Yes" to check all fields, "No" to uncheck all.'|trans({}, 'messages') }} <br>
+      {{ 'If you check a field, change its value, and save, the multistore behavior will not apply to this shop (or group), for this particular parameter.'|trans({}, 'messages') }}
```

Comment:
If this is a tooltip, please localize it in _Admin.Design.Help_. Otherwise in _Admin.Design.Feature_.